app/src/views/OperationalLearning/index.tsx

```diff
@@ -251,6 +219,13 @@ export function Component() {
         },
     });
 
+    // NOTE: This effect is intentionally run only once on the initial render
+    useEffect(() => {
+        triggerSummary(query);
+        triggerOperationalLearning(query);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Can't we not use Lazy request and trigger the summary and opsLearning query automatically on render?

Reply:
Yes, we are using `useLazyRequest`. This approach ensures we meet two primary cases: 
Case 1: `The API is triggered without any filter when the component initially render.`
Case 2: `When the user select and apply  filters, the API should be triggered again to reflect the new data based on the updated filters.`